#! /bin/sh /usr/share/dpatch/dpatch-run
## 78_kfbsd_tab.dpatch by Petr Salinger <Petr.Salinger@seznam.cz>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix build problem on kfreebsd

@DPATCH@
diff -urNad coreutils-7.2~/src/stty.c coreutils-7.2/src/stty.c
--- coreutils-7.2~/src/stty.c	2009-03-29 13:43:41.000000000 -0400
+++ coreutils-7.2/src/stty.c	2009-04-07 21:11:35.096288547 -0400
@@ -279,10 +279,18 @@
   {"cr0", output, SANE_SET, CR0, CRDLY},
 #endif
 #ifdef TABDLY
+#ifdef TAB3
   {"tab3", output, SANE_UNSET, TAB3, TABDLY},
+#endif
+#ifdef TAB2
   {"tab2", output, SANE_UNSET, TAB2, TABDLY},
+#endif
+#ifdef TAB1
   {"tab1", output, SANE_UNSET, TAB1, TABDLY},
+#endif
+#ifdef TAB0
   {"tab0", output, SANE_SET, TAB0, TABDLY},
+#endif
 #else
 # ifdef OXTABS
   {"tab3", output, SANE_UNSET, OXTABS, 0},
